lib/clients/index.ts

```diff
@@ -19,6 +20,7 @@ export interface ClientMap extends StaticType {
   metrics: MetricsType;
   mongo: MongoDB | null;
   pg: PostgresDB | null;
+  analyticsClient: AnalyticsSystem | null;
```

Comment:
```suggestion
  analyticsClient: AnalyticsSystem;
```
it seems like it's always being init (line 41 below), so no need for null

Reply:
done